api/config/cdr_versions_staging.json

```diff
@@ -22,6 +23,7 @@
     "releaseNumber": 1,
     "numParticipants": 946237,
     "cdrDbName": "synth_r_2019q1_2",
+    "elasticIndexBaseName": "synth_r_2019q1_2",
```

Comment:
So did you name this 'synth_r_2019q1_2' to be consistent with cdrDbName? Or was is the second version of the index?

Reply:
It was for consistency with cdrDbName. There's an argument to just use _1 here and make it correspond to the elastic index version instead. Don't feel too strongly on this (will probably be a follow-up if we want to change the convention).